src/gapi_lwt.ml

```diff
@@ -0,0 +1,13 @@
+let promise (f : _ -> unit) =
+  let (promise, resolver) = Lwt.task () in
+  f @@ Lwt.wakeup resolver ;
```

Comment:
```suggestion
  f @@ Lwt.wakeup resolver;
```

Reply:
Once again "ocaml-format" impose that seemingly useless space before the semi-colon.